lib/datadog/statsd/serialization/tag_serializer.rb

```diff
@@ -79,7 +79,7 @@ def escape_tag_content(tag)
         def dd_tags(env = ENV)
           return {} unless dd_tags = env['DD_TAGS']
 
-          to_tags_hash(dd_tags.split(','))
+          to_tags_hash(dd_tags.split(' '))
```

Comment:
To address concerns about backwards-compatibility, this could be changed to allow spaces or commas as the separator:

```suggestion
          to_tags_hash(dd_tags.split(/[, ]/))
```

Reply:
great idea: https://github.com/DataDog/dogstatsd-ruby/pull/289/commits/120c0d4a34b286b16ef08a7045c467d7115cf005

Reply:
The problem here is that if someone has spaces in their tag names then they will be split with this change. This could cause alerts to fire if they are monitoring a particular metric with a tag that has space in it.

Reply:
Yeah, I thought about that too. What about parsing it as CSV, including quote delimiters? It might slow things down - I'm not sure how often this gets called.

Reply:
And it would still require people to add quotes since they weren't there before.

Reply:
In the [DD docs](https://docs.datadoghq.com/getting_started/tagging/#define-tags) it seems like spaces are not supported in tag names.

Reply:
Spaces count as other special characters so will be converted to underscores once they are sent back to DD.

From the docs:

> Other special characters are converted to underscores.

Reply:
@carlosroman what if it's conditional - if DD_TAGS has a `,` in it, split on that. Otherwise split on spaces?

Reply:
https://github.com/DataDog/dogstatsd-ruby/pull/289/commits/48d0c527237fdd4096b97888fb24c12522726fc0